lib/core_ext/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -59,7 +63,8 @@ def add_index_with_concurrently(table_name, column_name, options = {})
         statements << "ON"
         statements << quote_table_name(table_name)
         statements << index_using          if index_using.present?
-        statements << "(#{index_columns})" if index_columns.present?
+        statements << "(#{index_columns})" if index_columns.present? unless skip_column_quoting
+        statements << "(#{column_name})"   if column_name.present? and skip_column_quoting
```

Comment:
Do you really mean `and` here? I rewatched http://www.virtuouscode.com/2014/08/26/how-to-use-rubys-english-andor-operators-without-going-nuts/ and I think this will logically work the same as `&&`, the `and` just seems odd when not using `and/or` for their special powers.

Reply:
The phrase "in the context of “Seattle-style” parenthesis-free code" encapsulates why. I love Ruby for its sparing use of syntax. Making code readable for PMs, and other interested non-coders makes a lot of business discussions easier to have. The style has become a habit for me. (I have no expectation here that anyone but engineers will read this code.)

Reply:
I get the attraction to use business user-friendly code, but the problem is if you become used to using and/or for boolean logic you could run into a situation in which the logic is incorrect:

```ruby
true || true && false
 => true 
true or true and false
 => false 
```

If you just stick to logical operators when performing logic and control flow operators when performing control flow, then you just can't make that particular mistake.

Reply:
Agreed that that is a big problem. (And that's a great, damning nutshell case.) Complex booleans (really anything with more than 2 terms) should not use the English operators.

Reply:
fwiw, I like `and/or` for control flow like suggested in the video. I'm not against them as a blanket rule.